help/en/package/jmri/jmrit/withrottle/UserInterface.shtml

```diff
@@ -62,45 +44,98 @@
       <img src="images/DigiTrainsScreen.png" height="120" width=
       "200" align="right" style="margin:20px;" alt=
       "DigiTrains for Android devices" title=
-      "DigiTrains for Android devices"></a></p>
-
-      <p>To use your iPhone, iPad or iPod Touch with JMRI as a
-      throttle, you run the <a href=
-      "http://withrottle.com">WiThrottle</a> app on the iPhone or
-      iPod Touch. The basic application is available for free
-      download from the iTunes store.</p>
-
-      <p>To use your Android device with JMRI as a throttle, you
-      run the <a href="http://enginedriver.mstevetodd.com">Engine
-      Driver</a> app or <a href=
-      "https://play.google.com/store/apps/details?id=hu.digitools.digitrains">
-      Digitrains</a> on the Android device. These free apps are
-      available from the <a href=
-      "https://play.google.com/store/apps/details?id=jmri.enginedriver">
-      Google Play Store</a> or the <a href=
-      "http://enginedriver.mstevetodd.com">EngineDriver site</a></p>
-
-      <p>You can connect from a Windows computer using the
-      <a href="https://www.microsoft.com/en-us/p/train-throttle/9nblggh4vf5j?activetab=pivot:overviewtab">Train Throttle Windows App</a>
-      available on the 
-      <a href="https://www.microsoft.com/en-us/p/train-throttle/9nblggh4vf5j?activetab=pivot:overviewtab">Microsoft store</a>.
+      "DigiTrains for Android devices"></a>
+   
+      
+      <h2>WiFi Throttle "applications" for your WiFi device</h2>
```

Comment:
> WiFi Throttle "applications" for your WiFi device

Only put quotes around a word when you're referring to the word itself. They're not a way of highlighting.

Reply:
should be updated now.